Lib/tarfile.py

```diff
@@ -57,13 +57,9 @@
     grp = None
 
 # os.symlink on Windows prior to 6.0 raises NotImplementedError
-symlink_exception = (AttributeError, NotImplementedError)
-try:
-    # OSError (winerror=1314) will be raised if the caller does not hold the
-    # SeCreateSymbolicLinkPrivilege privilege
-    symlink_exception += (OSError,)
-except NameError:
-    pass
+# OSError (winerror=1314) will be raised if the caller does not hold the
+# SeCreateSymbolicLinkPrivilege privilege
+symlink_exception = (AttributeError, NotImplementedError, OSError)
```

Comment:
This alias is now only used once in this file. You could just inline it here:

https://github.com/python/cpython/blob/67b4d2772c5124b908f8ed9b13166a79bbeb88d2/Lib/tarfile.py#L2278

Reply:
Someone out there might be using it. So, I went with the safest path

Reply:
Ugh, I guess so!